src/main/java/com/google/sps/servlets/PostCommentServlet.java

```diff
@@ -0,0 +1,103 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.sps.classes.SqlConstants;
+import com.google.sps.classes.Utility;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.util.Date;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** 
+ * This servlet will post a comment to an answer.
+ */
+@WebServlet("/post-comment")
+public class PostCommentServlet extends HttpServlet {
+
+  /** 
+   * This method will execute the query to insert a comment to the database.
+   */
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String body = request.getParameter("comment-body");
+    int questionId = Utility.tryParseInt(request.getParameter("question-id"));
+    int answerId = Utility.tryParseInt(request.getParameter("answer-id"));
```

Comment:
I see that these methods are called "tryParseInt" which makes me think that there's a chance it could fail? I just want to double check that fail states are accounted for here!

Reply:
yep, it had happened that we sent a null string or something while working on a feature and it would just return an error on parseInt, so this is to catch the error and log it